README.md

```diff
@@ -31,6 +31,38 @@ Once enabled the bundle will expose several services, such as:
 
 ## Event Store
 
+By default the [InMemoryEventStore](https://github.com/broadway/broadway/blob/master/src/Broadway/EventStore/InMemoryEventStore.php) is
+used.
+
+Broadway provides a persisting event store implementation using `doctrine/dbal`
+in [broadway/event-store-dbal](https://github.com/broadway/event-store-dbal).
+
+This can be installed using composer:
+
+```
+$ composer require broadway/event-store-dbal
+```
+
+You need to configure this event store in you application:
```

Comment:
`You will need to configure an event store in your application's service definition`?